py-polars/tests/unit/io/test_lazy_csv.py

```diff
@@ -220,7 +220,7 @@ def test_glob_skip_rows(tmp_path: Path) -> None:
 1,2,3
 4,5,6
 7,8,9
-    """
+"""
```

Comment:
This is where I deleted the extra 4 spaces that appear to only be present in the test function due to formatting.